components/feature/session/src/main/java/mozilla/components/feature/session/FullScreenFeature.kt

```diff
@@ -28,6 +29,7 @@ open class FullScreenFeature(
     }
 
     override fun onFullScreenChanged(session: Session, enabled: Boolean) = fullScreenChanged(enabled)
+    override fun onMetaViewportFitChanged(session: Session, viewportFit: String) = viewportFitChange(viewportFit)
```

Comment:
```suggestion
    override fun onFullScreenChanged(session: Session, enabled: Boolean) = fullScreenViewPortChanged(enabled, session.layoutInDisplayCutoutMode)
    override fun onMetaViewportFitChanged(session: Session, viewportFit: String) = (session.fullScreenMode, viewportFit)
```

See above for related change.

Reply:
Oh I just changed to from `onMetaViewportFitChange` to `onMetaViewportFitChanged` in the last patch to match the naming above.  Should I have not?

Reply:
Yeah if they are coupled together, then each call will be called once.  The consumer will then get both the full screen state and viewportFit value twice.

Reply:
The nit was on feature's constructor. :)

Reply:
Ah.  Thanks!